package.json

```diff
@@ -41,6 +42,10 @@
   },
   "devDependencies": {
     "@types/express": "^4.17.17",
-    "jshint": "^2.11.1"
+    "@types/jest": "^29.5.0",
+    "jest": "^29.5.0",
+    "jshint": "^2.11.1",
+    "node": "^19.8.1",
```

Comment:
We already have `ts-node` in the dependencies. Is there a reason why we need this (non-typescript node) for running tests?

Reply:
removed 'node' dependecy - 
<img width="621" alt="image" src="https://user-images.githubusercontent.com/22402315/229726558-565fa5a7-f2c2-43a9-8cfb-c2697d2f862b.png">

Reply:
Worked with @rohitkarhadkar, to solve this.